pkg/operation/botanist/cleanup.go

```diff
@@ -70,7 +70,7 @@ var (
 	GracePeriodFiveMinutes = utilclient.DeleteWith{client.GracePeriodSeconds(5 * 60)}
 
 	// NotSystemComponent is a requirement that something doesn't have the GardenRole GardenRoleSystemComponent.
-	NotSystemComponent = utils.MustNewRequirement(v1beta1constants.DeprecatedGardenRole, selection.NotEquals, v1beta1constants.GardenRoleSystemComponent)
```

Comment:
I guess we should keep this for a couple of releases to give people enough time to reconcile all their clusters, right? Also, some shoots might already have a deletion timestamp but still the deprecated label.

Reply:
As I tried to describe in the PR, the system components have also the no-cleanup label set via the ManagedResource and it should prevent them to be deleted during cluster deletion.

Reply:
OK, then we are good. Just wanted to raise and clarify this point again  to be sure we don't miss a case.